packages/icons/gulpfile.js

```diff
@@ -0,0 +1,52 @@
+import * as cheerio from 'cheerio';
+import { dest, src, task } from 'gulp';
+import concat from 'gulp-concat';
+import { gulpPlugin } from 'gulp-plugin-extras';
+import { basename } from 'path';
+
+function createCopyright() {
+  return `/**
+ * @license
```

Comment:
Now that we have a linter rule that adds a license header, we could omit it here, I guess.

Reply:
These are generated files so it's still good to have them. I closed the PR for now due to line-height problem.